components/director/pkg/inputvalidation/validators_cert_subject_mappings.go

```diff
@@ -36,6 +38,7 @@ var SupportedConsumerTypes = map[string]bool{
 	SuperAdminType:                         true,
 	BusinessIntegrationType:                true,
 	ManagedApplicationProviderOperatorType: true,
+	ManagedApplicationConsumerType:         true,
```

Comment:
This is used in the hydrator component, you should update the director version in the `hydrator` go.mod file